src/App.tsx

```diff
@@ -2,34 +2,34 @@ import React from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
+import { SearchField } from './components/SearchField';
 
-type State = {};
+type State = {
+  query: string;
+};
 
 export class App extends React.Component<{}, State> {
-  state: State = {};
+  state: State = {
+    query: '',
+  };
+
+  setQuery = (query: string) => {
+    this.setState({ query });
+  };
 
   render() {
+    const { query } = this.state;
+
+    const filteredMovies = [...moviesFromServer].filter(({ title, description }) => {
+      return description.includes(query) || title.toUpperCase().includes(query.toUpperCase());
```

Comment:
```suggestion
      return description.toUpperCase().includes(query.toUpperCase()) || title.toUpperCase().includes(query.toUpperCase());
```
I guess it would work better like this, so whenever some couch potato like me, who doesn't use uppercase that much often to google things, wants to find a movie which takes place in New York it would be easier for them

Reply:
Good point! Thanks! 🚀